### DISCUSSION TITLE
.quartoignore not visible in RStudio files panel

### DISCUSSION THREAD

**Discussion by andrewheiss** [quarto-dev/quarto-cli#2232] [Feature Requests]
I'm not sure if this is an RStudio issue or a Quarto issue. RStudio's Files panel generally hides files and directories that start with `.`, following Unix conventions. However, it does show hidden files that are related to R projects, like `.gitignore`, `.gitattributes`, `.Rhistory`, and so on. `.quartoignore` files don't show up in the list, though (RStudio on the left; Visual Studio Code on the right):

<img width="546" alt="image" src="https://user-images.githubusercontent.com/73663/187691390-505cae18-c0bd-4b15-93e7-0476c1494a13.png">

It's possible to change the settings of the Files panel to show all hidden files:

<img width="270" alt="image" src="https://user-images.githubusercontent.com/73663/187691574-80ca88f8-b387-4b96-9948-9e8b3c045347.png">

…but it might be nice to somehow treat `.quartoignore` files like other special R-specific hidden files like `.Rprofile` or `gitignore`

**Comment by mcanouil** [comment]
This is definitely an RStudio issue, since Quarto has nothing to do with the file manager of RStudio.
In my opinion, the file manager should not have some "R-specific behaviour" (if it is really a thing), because a project might have some other tools, etc.

**Reply by cderv** [reply]
This is indeed a RStudio IDE feature request. I added an issue there to track the idea: https://github.com/rstudio/rstudio/issues/11881
You can go add a  :+1:

**Reply by andrewheiss** [reply]
Awesome, thanks!

**Comment by cderv** [comment]
This should be now supported in RStudio IDE from 2022.11.0-daily+187 (see the resolved issue there for details). 

This will be present in a next major release of the IDE I believe. Thanks @andrewheiss for the report and idea!

**Comment by mcanouil** [comment]
Closing this as resolved https://github.com/rstudio/rstudio/issues/11881.
